main.groovy

```diff
@@ -272,6 +272,10 @@ def sendEmail(deploymentDirectories, updateType) {
             <td>Used Staging as Update</td>
             <td>${use_staging}</td>
         </tr>
+        <tr>
+            <td>Used APIM pre-release</td>
+            <td>${apim_pre_release}</td>
```

Comment:
Do we need to add this change for scenario tests as well?

Reply:
It will be in the future. I added it here just in case. Our original plan was to run both integration and scenario tests for pre-release versions

Reply:
Okay I'll remove this then

Reply:
Removed